cfme/tests/test_login.py

```diff
@@ -277,3 +126,55 @@ def test_multiregion_displayed_on_login(context, setup_multi_region_cluster, mul
         login_view = navigate_to(rapp.server, 'LoginScreen')
         assert login_view.is_displayed
         assert 'Remote' in login_view.details.region.text
+
+
+@pytest.mark.parametrize('context', [ViaUI])
+@pytest.mark.meta(blockers=[BZ(1632718)])
+@test_requirements.auth
+def test_update_password(context, request, appliance):
```

Comment:
Please mark this for browser isolation as well, because of its login/logout pattern.

Reply:
done